style.css

```diff
@@ -24,3 +24,12 @@ img {
 h1 {
   color: #ffe3dc;
 }
+
+.hello {
+  font-family: "Josefin Sans", sans-serif;
+  font-size: 100px;
+}
+
+.headertext {
+  color: #ffe3dc;
+}
```

Comment:
You've set this property with the same value somewhere else already, might as well have a reusable class for that